test.js

```diff
@@ -10,4 +10,9 @@ test(async t => {
 	t.false(await fn(notClean));
 	t.true(fn.sync(clean));
 	t.false(fn.sync(notClean));
+	t.false(await fn(notClean, {ignore: ['dirty.txt']}));
+	t.true(await fn(notClean, {ignore: ['dirty.txt', 'dirtier.txt']}));
+	t.true(await fn(notClean, {files: ['dirty.txt'], ignore: ['dirty.txt']}));
+	t.true(await fn.sync(notClean, {ignore: ['dirty.txt', 'dirtier.txt']}));
+	t.true(await fn.sync(notClean, {files: ['dirty.txt'], ignore: ['dirty.txt']}));
```

Comment:
Just drop the `ignore` option. People can use the `!` operator from `multimatch`.

So, instead of

```js
{
  files: ['dirt*.txt'],
  ignore: ['dirtier.txt']
}
```

You just do:

```js
{
  files: ['dirt*.txt', '!dirtier.txt']
}  
```